terminator/config.go

```diff
@@ -21,15 +21,15 @@ type Config struct {
 // String impl.
 func (c Config) String() string {
 	return utils.StructString(c, map[string]func(interface{}) interface{}{
-		"SOURCE_SECRET_SEED":  utils.SecretKey2PublicKey,
-		"TRADING_SECRET_SEED": utils.SecretKey2PublicKey,
+		"SourceSecretSeed":  utils.SecretKey2PublicKey,
```

Comment:
If we were to run terminator currently then it would log out the secret keys because this transformation would not run.

Since `StructString` changed to take in a map keyed by the `feedDisplayName` this should be changed back to `SOURCE_SECRET_SEED` and `TRADING_SECRET_SEED` instead of using variable names.
This also applies to the `String()` method in `trader/config.go` which currently runs because the map it uses is keyed by the correct values (and is inconsistent with this method).

I'm fine with all these functions using either the _variableName_ or the _CONFIG_NAME_ as long as they are all consistent and run the transformations correctly.